ExceptionsHomework/src/ro/scit/entities/Student.java

```diff
@@ -0,0 +1,91 @@
+package ro.scit.entities;
+import ro.scit.utilities.DateOfBirth;
+import java.util.Objects;
+
+/**
+ * @author Georgiana
+ * Student class
+ */
+
+public class Student {
+    //    private static int nextId = 1;
```

Comment:
As a best practice, we shouldn't leave commented out lines in the codebase when pushing our work. If you need to go back to some piece of code after you've deleted something, Git remembers everything and you can access all the history of your code.